mint.json

```diff
@@ -33,14 +33,27 @@
   "primaryTab": {
     "name": "Cobo Portal"
   },
-  "tabs": [
+  "anchors": [
     {
       "name": "Cobo Accounts",
-      "url": "cobo-accounts"
+      "url": "cobo-accounts",
+      "icon": "users"
     },
     {
       "name": "Cobo Guard",
-      "url": "cobo-guard"
+      "url": "cobo-guard",
+      "icon": "shield-halved"
+    },
+    {
+      "name": "Access Private Content",
+      "url": "https://cobo-oauth.vercel.app/private",
```

Comment:
This will change to `manuals.cobo.com/private` or `cobo.com/manuals/private` once we update the official URL